shared/elements/gaia_lowstorage_dialog/examples/locales/example.en-US.properties

```diff
@@ -0,0 +1,7 @@
+lowstorage-dialog-title=Low Device Storage
+lowstorage-dialog-generic-text=At least 10 MB of space is required. Delete apps and media files, then restart your device.
+lowstorage-dialog-learnmore=Learn More
+lowstorage-dialog-ok=OK
+
+lowstorage-explanation=Less than 5MB remain.
+lowstorage-explanation-details=Messages can not be sent, saved or received, not can Contacts be added at this time.
```

Comment:
this will need to be copied in the apps that use the component; maybe I should put this in a README.